### ISSUE TITLE
Support for DAS?

### ISSUE THREAD

**Issue by Thanh-Binh** [inilabs/caer#14]
does the current cAER support capturing data from DAS and reading recorded data back for offline processing?
Thanks

**Comment by llongi** [comment]
No, we currently don't support DAS in cAER/libcaer. Support is planned at some point in the future.

**Comment by Thanh-Binh** [comment]
thanks

**Comment by llongi** [comment]
Depends on libcaer/#1.
